modules/understanding-upgrade-channels.adoc

```diff
@@ -81,6 +81,7 @@ ifndef::openshift-origin[]
 In addition to the stable channel, certain minor versions of {product-title} offer an link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[Extended Update Support] (EUS). These EUS versions extend the maintenance phase for customers with Premium Subscriptions to 14 months. {product-title} 4.6 is currently the only minor version with EUS.
 
 Although there is no difference between stable-4.6 and eus-4.6 channels until {product-title} 4.6 transitions to the EUS phase, you can switch to the EUS channel as soon as it becomes available. When {product-title} 4.6 transitions to the EUS phase of its lifecycle, the stable-4.6 channel will no longer receive subsequent z-stream updates. After you upgrade to a version that is exclusive to the EUS channel, that cluster will no longer be eligible for minor version upgrades until upgrades to the next EUS version become available. The next planned EUS version is to be 4.10 and the upgrade to that version will require a serial set of version upgrades, such as from 4.6 to 4.7 to 4.8 to 4.9 to 4.10.
+//Does this need an update?
```

Comment:
@kmccarron-rh, is this paragraph still accurate?

Reply:
@LalatenduMohanty  Would you review and approve this change for OpenShift 4.9 GA?